pkgs/development/tools/skopeo/default.nix

```diff
@@ -28,28 +25,26 @@ let
 
   goPackagePath = "github.com/containers/skopeo";
 
+  vendorPath = "${goPackagePath}/vendor/github.com/containers/image/v5";
+
 in
 buildGoPackage {
   pname = "skopeo";
   inherit version;
-  inherit src goPackagePath;
+  inherit src goPackagePath vendorPath;
 
   outputs = [ "bin" "man" "out" ];
 
-  excludedPackages = "integration";
+  excludedPackages = [ "integration" ];
 
-  nativeBuildInputs = [ pkg-config (lib.getBin go-md2man) ];
-  buildInputs = [ gpgme ] ++ lib.optionals stdenv.isLinux [ libgpgerror lvm2 btrfs-progs libselinux ];
+  nativeBuildInputs = [ pkg-config go-md2man ];
```

Comment:
I think you are increasing the dependency closure size since you use `go-md2man.out` while we were using `go-md2man.bin` before.

Reply:
Doesn't seem to change the size? `out` is empty.

I'm changing this for consistency as it isn't used anywhere else with `go-md2man` and I don't think using `lib.getBin` in `nativeBuildInputs` is correct?

Reply:
Yeah. Should be fine.

Reply:
We now have `strictDeps = true;` on. So this might not even matter anymore.